safe-global/web-core
# Fix: CSS-only dark mode

## What it solves

Resolves #804 

## How this PR fixes it
Makes the dark CSS vars kick in before the theme is initialized.
However, if the light mode is forced via the user toggle, they'll be overridden.

## How to test it
Switch the OS to dark mode and load/reload the site. It should not flash the light theme.
Switch the dark mode toggle to light and try the same.
